samples/system-test/automlTranslation.test.js

```diff
@@ -0,0 +1,147 @@
+
+const cmdDataset = `node ./automl/automlTranslationDataset.js`;
+const cmdModel = `node ./automl/automlTranslationModel.js`;
+const cmdPredict = `node ./automl/automlTranslationPredict.js`;
+
+// const flowerTrainData = `gs://`;
+// const cwd = path.join(__dirname, `..`);
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = 'dummyModel';
+const sampleText = `./automl/resources/testInput.txt`;
+
+const donotdeleteModelId = `TRL188026453969732486`;
+//const donotdeleteDatasetId = `TRL2551826603472450019`;
+
+// skipped because it's been taking too long to delete datasets
+test.skip(`It should create a create, list, and delete a dataset`, async t => {
+  // Check to see that this dataset does not yet exist
+  let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+  t.false(output.includes(testDataSetName));
+
+  // Create dataset
+  output = await tools.runAsync(
+    `${cmdDataset} createDataset -n "${testDataSetName}"`
+  );
+  const dataSetId = output
+    .split(`\n`)[1]
+    .split(`:`)[1]
+    .trim();
+  t.true(output.includes(`Dataset display name:  ${testDataSetName}`));
+
+  // delete dataset
+  output = await tools.runAsync(
+    `${cmdDataset} deleteDataset -i "${dataSetId}"`
+  );
+  t.true(output.includes(`Dataset deleted.`));
+});
+
+// we make too models running this test, see hard-coded workaround below
+test.skip(`It should create a dataset, import data, and start making a model`, async t => {
+  // Check to see that this dataset does not yet exist
+  let output = await tools.runAsync(`${cmdDataset} listDatasets`);
+  t.false(output.includes(dummyDataSet));
+
+  // Create dataset
+  output = await tools.runAsync(
+    `${cmdDataset} createDataset -n "${dummyDataSet}"`
+  );
+  const dataSetId = output
+    .split(`\n`)[1]
+    .split(`:`)[1]
+    .trim();
+  t.true(output.includes(`Dataset display name:  ${dummyDataSet}`));
+
+  // Import Data
+  output = await tools.runAsync(
+    `${cmdDataset} importData -i "${dataSetId}" -p "gs://nodejs-docs-samples-vcm/flowerTraindata20lines.csv"`
+  );
+  t.true(output.includes(`Data imported.`));
+
+  // Check to make sure model doesn't already exist
+  output = await tools.runAsync(`${cmdModel} listModels`);
+  t.false(output.includes(`${testModelName}`));
+
+  // begin training dataset, getting operation ID for next operation
+  output = await tools.runAsync(
+    `${cmdModel} createModel -i "${dataSetId}" -m "${testModelName}" -t "2"`
+  );
+  const operationName = output
+    .split(`\n`)[0]
+    .split(`:`)[1]
+    .trim();
+  t.true(output.includes(`Training started...`));
+
+  // poll operation status, here confirming that operation is not complete yet
+  output = await tools.runAsync(
+    `${cmdModel} getOperationStatus -i "${dataSetId}" -o "${operationName}"`
+  );
+  t.true(output.includes(`done: false`));
+});
+
+test(`It should run get model (from a prexisting model)`, async t => {
+  //confirm dataset exists
```

Comment:
Über-nit: space after `//`, and capitalize first letter.

Apply this comment throughout your PR.